ps_emailsubscription.php

```diff
@@ -97,6 +97,7 @@ public function install()
                     'displayFooterBefore',
                     'actionCustomerAccountAdd',
                     'additionalCustomerFormFields',
+                    'displayAdminCustomersForm',
```

Comment:
The permissions for this file have been modified
![screen shot 2017-08-17 at 11 31 06](https://user-images.githubusercontent.com/1009343/29405886-9676eb1a-833f-11e7-990e-0bfbd7ca72ca.png)